Modules/_json.c

```diff
@@ -1308,27 +1311,30 @@ _encoded_const(PyObject *obj)
 {
     /* Return the JSON string representation of None, True, False */
     if (obj == Py_None) {
-        static PyObject *s_null = NULL;
+        _Py_static_string(PyId_null, "null");
+        PyObject *s_null = _PyUnicode_FromId(&PyId_null);
         if (s_null == NULL) {
-            s_null = PyUnicode_InternFromString("null");
+            return NULL;
         }
-        Py_XINCREF(s_null);
+        Py_INCREF(s_null);
         return s_null;
```

Comment:
```suggestion
        return Py_NewRef(s_null);
```

Same remark for code below.